src/main/java/resources/config.example

```diff
@@ -0,0 +1,3 @@
+*Please create a config.properties file (default path: src/main/java/resources/config.properties) and define paths to both webdrivers as shown below:
```

Comment:
Please add similar instruction to readme as well.

Reply:
@Fishgame1 Instructions will be added to README file as part of the singleton implementation task as well.